app/javascript/src/assets/scss/widgets/_tabs.scss

```diff
@@ -1,50 +1,52 @@
 .tabs {
-  border-width: 0;
   @include padding($zero $space-normal);
   @include border-normal-bottom;
+  border-width: 0;
```

Comment:
This would override border-normal-bottom, right?

Reply:
Yes. I will check it and make commit if need.

Reply:
I've fixed it by extended border-width properties